docker/slurm/docker-compose.yaml

```diff
@@ -0,0 +1,77 @@
+version: "3.8"
+
+services:
+
+  mysql:
+    image: mysql:5.7
+    hostname: mysql
+    container_name: mysql
+    environment:
+      MYSQL_RANDOM_ROOT_PASSWORD: "yes"
+      MYSQL_DATABASE: slurm_acct_db
+      MYSQL_USER: slurm
+      MYSQL_PASSWORD: password
+    volumes:
+      - var_lib_mysql:/var/lib/mysql
+
+  slurmdbd:
+    image: grid-engine-api-slurm-base
+    build:
+      context: ../..
+      dockerfile: ./docker/slurm/Dockerfile.base
+    command: [ "slurmdbd" ]
+    container_name: slurmdbd
+    hostname: slurmdbd
+    volumes:
+      - etc_munge:/etc/munge
+      - etc_slurm:/etc/slurm
+      - var_log_slurm:/var/log/slurm
+    expose:
+      - "6819"
+    depends_on:
+      - mysql
+
+  slurmctld:
+    image: grid-engine-api-slurm-base
+    command: [ "slurmctld" ]
+    container_name: slurmctld
+    hostname: slurmctld
+    volumes:
+      - etc_munge:/etc/munge
+      - etc_slurm:/etc/slurm
+      - slurm_jobdir:/data
+      - var_log_slurm:/var/log/slurm
+    expose:
+      - "6817"
+    depends_on:
+      - "slurmdbd"
+
+  c1:
```

Comment:
I'm facing some problems during that service startup, in case of the first build (no images are presented on the host):
```log
#5 [grid-engine-api-slurm-service internal] load metadata for docker.io/library/grid-engine-api-slurm-base:latest
#5 sha256:b2b63de517b1d22c3bb2588f7e1f8cc494b45f34f8aeb324e4c5e79e2dd29269
#5 ERROR: pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed
```
It looks like `base` image is not available at the `c1` build time, so it is being pulled from the remote repo, where it doesn't really exist. 

Can you, please, try to reproduce it:
1. Remove all the pre-built images from your host
2. Run `compoesUp` task
3. Check if any problems occurs

In case there are any issues, please, take a closer look on how we could fix them:
- explicit `composeBuild` call should be configured before executing `composeUp`
- it might be more clear to build Slurm-related images in advance outside the containers' compose process with raw `docker build` calls